### PR TITLE
My experience with homework 1

### PR DESCRIPTION
**Pull request by sharanyabashyam**
**Hello, my name is Sharanya.** 

Editing the readme document was the easiest of all. I just followed the Rcheatsheet. I had to google to check how I could preview if the commands that I typed were right or wrong, and that is how I discovered the preview button on the R console. 

I face some issues in embedding an image on the file. That took a while because initially, only the link of the image was showing. Again, googling helped, because it directed me to blogs where people had faced a similar issue. I ended up using an image of mine that was publically available on the internet. It also took me a while to install git, and finish all the steps, but that was more because my laptop ran out of battery midway and I didn't notice until it shut down.

Honestly, in the beginning, when I saw the assignment for the first time I thought it was something very complicated and I wouldn't be able to do it alone. Now that it is done, and I figured out how to run do the entire workflow process (the video with the homework workflow was very useful, thank you for that Dr.Soltoff!), I feel more confident in using Markdown and more comfortable with it.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,30 @@
-# hw01
+# HOMEWORK 1 #
+# **About Me** #
```
Comment (djanvier) [inline README.md:2]: Headers are automatically bolded, so don't need to include that syntax here

Comment (djanvier) [reply README.md:2]: You only need the hash sign at the beginning of the line
See below:

`# HOMEWORK 1 #`
# HOMEWORK 1 # 
 
` # HOMEWORK 1`
# HOMEWORK 1

File: README.md [hunk 2]
```diff
@@ -1 +1,30 @@
...
+
+![A picture of me](https://cpb-us-w2.wpmucdn.com/voices.uchicago.edu/dist/7/1535/files/2020/07/Sharanya.jpg)
```
Comment (djanvier) [inline README.md:4]: Nice!

File: README.md [hunk 3]
```diff
@@ -1 +1,30 @@
...
+* Reading books (At present, I am reading *Killing Commendatore* by Haruki Murakami )
+* Music (Currently I am obsessed with the *Folklore* album, *If I left the world by Gryffin* and *Dynamite* by BTS).
+    + Some of my favourite groups/singers:
+      - BTS 
+      - ABBA
+      - ALEPH
+      - keshi
+      - Taylor Swift
+      - Halsey
+      - TXT
```
Comment (djanvier) [inline README.md:20]: Great unordered list!

**Review by djanvier — COMMENTED** [review]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Improvement |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |           |       x       |            |
| **Coding strategy**         |           |      x        |            |
| **Presentation**            |    x       |              |            |
| **Achievement, creativity** |     x      |              |            |
| **Reproducibility**         |           |       x       |            |
 
- Great job overall!
- Just make sure you are committing **early and often** and that you are writing meaningful commit message (ex: a separate commit message for the image file)

**Review by jessezorr — COMMENTED** [review]
Hi Sharanya!

Your bio is good.  I liked the variety of different bullet points you were able to use, and good placement of the picture.  Good work!

Jesse

**Comment by sharanyabashyam** [comment]
> 
> 
> Hi Sharanya!
> 
> Your bio is good. I liked the variety of different bullet points you were able to use, and good placement of the picture. Good work!
> 
> Thank you Jesse !

**Pull request closed without merging**